extensions/amp-script/0.1/amp-script.js

```diff
@@ -291,6 +311,7 @@ export class AmpScript extends AMP.BaseElement {
       onReceiveMessage: (data) => {
         dev().info(TAG, 'From worker:', data);
       },
+      sandbox: this.sandboxed_ && {iframeUrl},
```

Comment:
What uses this?

Reply:
Nothing yet, but soon to be Permutive: https://github.com/ampproject/amphtml/issues/30193